README.md

```diff
@@ -1 +0,0 @@
-# portfolio-site
```

Comment:
You'll eventually want to add a README here to display if anyone opens up this github page but it isn't strictly necessary right now